neptune/new/internal/operation_processors/async_operation_processor.py

```diff
@@ -159,6 +168,11 @@ def _wait_for_queue_empty(self, initial_queue_size: int, seconds: Optional[float
                 )
                 return
 
+            if not self._consumer.is_running():
+                exception = NeptuneSynchronizationAlreadyStoppedException()
+                logger.warning(str(exception))
+                return
```

Comment:
Do we need to additionally print `NeptuneSynchronizationAlreadyStoppedException()` here?
We're running `flush` only from `wait`, where we already check consumer for running and raise exception if necessary.

Reply:
What does it have to do with flush? It's in `_wait_for_queue_empty` called on stop().

Reply:
> What does it have to do with flush?

Eh, sorry, I've analyzed the code in github...